pkgs/data/fonts/victor-mono/default.nix

```diff
@@ -10,20 +10,30 @@ in fetchFromGitHub {
   repo = pname;
   rev = "v${version}";
 
+  # Upstream prefers we download from the website,
+  # but we really insist on a more versioned resource.
+  # Happily, tagged releases on github contain the same
+  # file `VictorMonoAll.zip` as from the website,
+  # so we extract it from the tagged release.
+  # Both methods produce the same file, but this way
+  # we can safely reason about what version it is.
   postFetch = ''
-    tar xf $downloadedFile --strip=1
-    unzip public/VictorMonoAll.zip TTF/\*
-    mkdir -p $out/share/fonts/truetype/${pname}
-    cp TTF/*.ttf $out/share/fonts/truetype/${pname}
+  tar xvf $downloadedFile --strip-components=2 --wildcards \*/public/VictorMonoAll.zip
+
+  mkdir -p $out/share/fonts
+
+  unzip -j VictorMonoAll.zip \*.ttf -d $out/share/fonts/truetype
```

Comment:
> don't put fonts in ${pname} subdirs
fontconfig's hashing/cache is mostly geared
for the many-fonts-in-one-place situation,
although this may be fixed in more recent versions

IIRC that was a really long time ago and I think lots of distros that do that stuff globally just use subdirs